File: test/cases/record/test_record_if_condition.pas
program test;
type
  person = record
    age: integer;
    score: real;
    active: boolean
  end;
var
  p: person;

begin
  p.age := 25;
  p.score := 85.5;
  p.active := true;

  if p.active then
  begin
    if p.score > 90.0 then
      write(1)
    else
      write(0)
  end
  else
    write(-1)
end.

File: test/cases/record/test_record_if_condition.c
#include <stdio.h>
typedef struct {
    int age;
    float score;
    int active;
} person;
person p;


int main(void) {
    p.age = 25;
    p.score = 85.5;
    p.active = 1;
    if (p.active) {
        if ((p.score > 90.0)) {
            printf("%d", 1);
        } else {
            printf("%d", 0);
        }
    } else {
        printf("%d", (-1));
    }

    return 0;
}
